tests/test_preproc.py

```diff
@@ -603,6 +603,21 @@ def test_calc_eye_calibration(self):
 
         np.testing.assert_allclose(expected_coeff, coeff)
         np.testing.assert_allclose(expected_corr_coeff, corr_coeff)
+        
+    def test_calc_eye_target_calibration(self):
+        eye_data = np.array([[0, 1, 2, 3, 4, 5, 6, 7], [3, 4, 5 ,6 ,7, 8, 9, 10]]).T
+        eye_samplerate = 1
+        event_times = [2, 3, 4, 5]
+        event_codes = [0, 1, 0, 1]
+        target_pos_test = np.array([[2,5,0],[4,7,0]])
```

Comment:
in the real data you won't have a target position for every event. i suggest adding some irrelevant event codes to your test. e.g. event_times = [2, 3, 4, 5, 6, 7, 8] and event_codes = [0, 2, 1, 3, 0, 5, 1]